source/includes/usage-examples/index-code-examples.cpp

```diff
@@ -0,0 +1,18 @@
+// start-single-field
+auto index_specification = make_document(kvp("<field name>", 1));
```

Comment:
```suggestion
auto index_specification = make_document(kvp("title", 1));
```

Suggest using "title" to create index ("title_1") referenced in subsequent examples.

Reply:
By convention we use placeholders on these landing pages (formerly known as "usage example" pages) instead of giving actual fields so that the user can copy/paste these and easily see where to swap out for their specific use case

Reply:
Got it. In that case, consider updating the example below to replace `title_1` with `<field name>_1`.

Reply:
Oops. Good catch, thanks!